client/app/telecom/telephony/service/fax/filtering/telecom-telephony-service-fax-filtering.html

```diff
@@ -112,42 +107,43 @@ <h2 data-translate="loading"></h2>
                         data-translate="telephony_service_fax_filtering_config">
                     </h2>
                 </header>
-                <div class="clearfix bottom-space-m8">
-                    <div class="pull-left">
-                        <button
-                            type="button"
-                            class="btn btn-default"
-                            data-ng-disabled="$ctrl.getSelection().length === 0 || $ctrl.screenLists.isDeleting"
-                            data-ng-click="$ctrl.removeSelectedScreenLists()">
-                            <span class="ovh-font ovh-font-bin line-height-normal right-space-m4" aria-hidden="true"></span>
+                <div class="clearfix mb-2">
+                    <div class="float-left">
+                        <button type="button"
+                                class="btn btn-default"
+                                data-ng-disabled="$ctrl.getSelection().length === 0 || $ctrl.screenLists.isDeleting"
+                                data-ng-click="$ctrl.removeSelectedScreenLists()">
+                            <span class="ovh-font ovh-font-bin line-height-normal mr-1"
+                                  aria-hidden="true"></span>
                             <span data-ng-if="$ctrl.getSelection().length <= 1"
                                   data-translate="telephony_service_fax_filtering_table_delete">
                             </span>
                             <span data-ng-if="$ctrl.getSelection().length > 1"
                                   data-translate="telephony_service_fax_filtering_table_delete_plural"
                                   data-translate-values="{ nb: $ctrl.getSelection().length }">
                             </span>
-                            <spinner class="left-space-m4" data-ng-if="$ctrl.screenLists.isDeleting"></spinner>
+                            <bouncing-box-loader class="d-inline-block align-middle ml-2"
```

Comment:
Replace `ovh-font-bin` icon with the `bouncing-box-loader ` when `$ctrl.screenLists.isDeleting` is true instead?